### PR TITLE
Aa/rn upgrade

### PR DESCRIPTION
**Pull request by Nikituh**

### CODE REVIEW

**Comment by hudson-pierce** [comment]
run tests

File: App.tsx [hunk 1]
```diff
@@ -217,28 +215,29 @@ export class App extends React.Component {
...
                   keyExtractor={item => item.id}
+                  contentInsetAdjustmentBehavior={"automatic"}
               />
-            </ScrollView>
           </SafeAreaView>
 
           <Overlay visible={this.state.barcodeModalVisible} style={styles.overlay} onClose={this.onSave} closeOnTouchOutside>
             <Text style={styles.subtitle}>ACCEPTED BARCODE TYPES</Text>
             <BarcodeList/>
+            {/*@ts-ignore*/}
```
Comment (marcosaia1812) [inline App.tsx:233]: Why do we need these ignore directives?

**Comment by hudson-pierce** [comment]
run tests

**Comment by hudson-pierce** [comment]
run tests

**Comment by hudson-pierce** [comment]
run tests

**Review by marcosaia1812 — APPROVED** [review]

**Pull request merged**